src/app/routes/enrolment/enrolment-list/enrolment-list.component.ts

```diff
@@ -139,8 +139,9 @@ export class EnrolmentListComponent implements OnInit {
 
   private async appendDidDocSyncStatus(list: any[]) {
     // Get Approved Claims in DID Doc & Idenitfy Only Role-related Claims
-    let claims: any[] = await this.iamService.iam.getUserClaims();
-    claims = claims.filter((item: any) => {
+    const did = this.listType === EnrolmentListType.ASSET ? { did: this.subject } : undefined;
+    const claims: any[] = (await this.iamService.iam.getUserClaims(did))
```

Comment:
Just curious, why the explicit setting to `any[]` here? Could you just use type that is inferred from the return of `getUserClaims()`?

Reply:
Updated.